crates/edr_evm/src/runtime.rs

```diff
@@ -66,6 +66,7 @@ where
 {
     cfg.disable_balance_check = true;
     cfg.disable_block_gas_limit = true;
+    cfg.disable_nonce_check = true;
```

Comment:
This disables nonce checks for all calls; i.e `eth_call`, `debug_traceCall`, and `eth_estimateGas`. Is this desired @fvictorio ?

Reply:
I think it's desired, yes. I also checked and this matches geth's behavior.